### PR TITLE
initial commit of Upload Config File functionality

### PR DESCRIPTION
**Pull request by mbay-SAG**

### CODE REVIEW

**Comment by mbay-SAG** [comment]
Ding download into same branch

**Comment by mbay-SAG** [comment]
done, please review!

File: c8ydm/agentmodules/configfile_download_handler.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        self.agent.publishMessage(success)
+
+    def _set_failed(self, reason):
+        failed = SmartRESTMessage('s/us', '502', [self.fragment,reason])
+        self.logger.error(f'Operation failed, reason: {reason}')
+        self.agent.publishMessage(failed)
+    
+    def handleOperation(self, message):
+        mo_id = self.agent.rest_client.get_internal_id(self.agent.serial)
+        configfiles = {'sshd': '/etc/ssh/sshd_config', 'agent': '/root/.cumulocity/agent.ini'}
```
Comment (switschel) [inline c8ydm/agentmodules/configfile_download_handler.py:59]: Replace /root/ with Userfolder.

File: c8ydm/agentmodules/configfile_upload_handler.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        self.agent.publishMessage(success)
+
+    def _set_failed(self, reason):
+        failed = SmartRESTMessage('s/us', '502', [self.fragment,reason])
+        self.logger.error(f'Operation failed, reason: {reason}')
+        self.agent.publishMessage(failed)
+    
+    def handleOperation(self, message):
+        mo_id = self.agent.rest_client.get_internal_id(self.agent.serial)
+        configfiles = {'sshd': '/etc/ssh/sshd_config', 'agent': '/root/.cumulocity/agent.ini'}
```
Comment (switschel) [inline c8ydm/agentmodules/configfile_upload_handler.py:59]: Replace /root/ with userfolder.

**Review by switschel — COMMENTED** [review]
Please fix hard coded /root/ and replace it with user folder.

File: c8ydm/agentmodules/configfile_upload_handler.py (hunk 2 above)
Comment (mbay-SAG) [reply c8ydm/agentmodules/configfile_upload_handler.py]: corrected

File: c8ydm/agentmodules/configfile_download_handler.py (hunk 1 above)
Comment (mbay-SAG) [reply c8ydm/agentmodules/configfile_download_handler.py]: corrected!

**Comment by mbay-SAG** [comment]
Done, please review!

**Pull request merged**